packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -80,7 +75,7 @@ export class Image extends BaseComponent<IImageProps, IImageState> {
     let imageProps = getNativeProps(this.props, imageProperties, ['width', 'height']);
     let { src, alt, width, height, shouldFadeIn, className, imageFit, role, maximizeFrame } = this.props;
     let { loadState } = this.state;
-    let coverStyle = this._coverStyle;
+    let coverStyle = this.props.coverStyle || this._coverStyle;
```

Comment:
This should probably be 

```ts
this.props.coverStyle === undefined ? this._coverStyle : this.props.coverStyle
```
I think the intent is that if the caller passes in a cover style, we use that. But I think the behavior is, if they pass in landscape (value 0), use `_coverStyle`.